### PR TITLE
Feature/connect auth

### PR DESCRIPTION
**Pull request by DemogorGod**
connected auth flow and fixed shifting on modal opening.

### CODE REVIEW

**Comment by ccali11** [comment]
Taking a look now.

**Comment by ccali11** [comment]
I think "dropdown" is the wrong component type at the top of the first wallet modal:

image

**Comment by ccali11** [comment]
As a user, I would expect instead of the addresses appearing within the dropdown for it to "slide" to the next screen within that modal and list the available addresses as well as a back button to go back and select a different provider.

**Comment by ccali11** [comment]
There is a small bug where after connecting a first wallet address (say with provider MetaMask), then when I go to select Ledger, for a moment the MetaMask address still shows.

**Comment by ccali11** [comment]
## User Interface Feedback
Looking great!

## User Experience Feedback
- [ ] As a user, I would expect the "Select Provider" modal to completely change to a "Select Address" modal instead of having a dropdown above the providers.
- [ ] As a user, I would like a way of knowing that I'm confirmed signed up or logged in or that my address was added to my user account
- [ ] As a user, I would like a way of knowing if I'm currently logged in or not (this could possibly be solved with however we solve the above point about confirming user successfully signed up, logged in, or added an address to their user account)
- [ ] small bug where after connecting a first wallet address (say with provider MetaMask), then when I go to select Ledger, for a moment the MetaMask address still shows

File: apps/web/index.html [hunk 1]
```diff
@@ -12,9 +12,7 @@
     
     
     <!-- Icon Library -->
-    <!-- <script src="path/to/dist/feather.js"></script> -->
-    <!-- <script src="https://unpkg.com/feather-icons"></script> -->
-    <script src="https://unpkg.com/feather-icons"></script>
+    <script src="https://cdn.jsdelivr.net/npm/feather-icons/dist/feather.min.js"></script>
```
Comment (ccali11) [inline apps/web/index.html:15]: Does this mean we don't need the feather package in our `package.json` file?

File: apps/web/src/App.vue [hunk 2]
```diff
@@ -2,17 +2,17 @@
   import useUsers from '@/composables/users'
   import { RouterView } from 'vue-router'
   import DefaultLayout from '@/layouts/default-layout.vue'
-  import { onMounted } from 'vue'
+  // import { onMounted } from 'vue'
```
Comment (ccali11) [inline apps/web/src/App.vue:5]: Can we just delete all commented out code? Still need `user` from `useUsers` here?

File: apps/web/src/composables/ethers.ts [hunk 3]
```diff
@@ -259,6 +260,10 @@ export default function useEthers() {
 
 function getBrowserProvider(providerString: ProviderString) {
   if (providerString === 'MetaMask' || providerString === 'CoinbaseWallet') {
+    const { ethereum } = window
```
Comment (ccali11) [inline apps/web/src/composables/ethers.ts:263]: I think we did this together, right? 👍

**Review by ccali11 — CHANGES_REQUESTED** [review]
## User Interface Feedback
Looking great!

## User Experience Feedback
- [ ] As a user, I would expect the "Select Provider" modal to completely change to a "Select Address" modal instead of having a dropdown above the providers.
- [ ] As a user, I would like a way of knowing that I'm confirmed signed up or logged in or that my address was added to my user account
- [ ] As a user, I would like a way of knowing if I'm currently logged in or not (this could possibly be solved with however we solve the above point about confirming user successfully signed up, logged in, or added an address to their user account)
- [ ] Small bug where after connecting a first wallet address (say with provider MetaMask), then when I go to select Ledger, for a moment the MetaMask address still shows

File: apps/web/index.html (hunk 1 above)
Comment (DemogorGod) [reply apps/web/index.html:15]: Yeah, I removed that file as well.

File: apps/web/src/App.vue (hunk 2 above)
Comment (DemogorGod) [reply apps/web/src/App.vue:5]: Will do!

**Review by ccali11 — APPROVED** [review]

**Pull request merged**